Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Cancel.hs

```diff
@@ -290,11 +290,18 @@ driverDistanceToPickup booking merchantOperatingCityId tripStartPos tripEndPos =
 
 getCancellationDuesDetails :: (Id Person.Person, Id Merchant.Merchant) -> Flow CancellationDuesDetailsRes
 getCancellationDuesDetails (personId, merchantId) = do
```

Comment:
Instead of fetching the booking, we can pass bookingId from UI itself?

Reply:
Fixed